packages/ilp-spsp-payout/src/lib/PayoutConnection.ts

```diff
@@ -177,8 +176,8 @@ export class PayoutConnection {
     const onClose = () => cleanUp()
     const onError = () => cleanUp()
     const onOutgoingMoney = (amount: string) => {
-      this.totalStreamAmount += Number(amount)
-      if (this.totalStreamAmount + this.sent === this.target) {
+      totalStreamAmount += Number(amount)
+      if (totalStreamAmount + this.sent === this.target) {
```

Comment:
I think this should be `>= target`, not `===`. The code here is using javascript `Number`s instead of bignums or `Long`s -- float rounding might make it miss the exact value.